xbmc/interfaces/legacy/ModuleXbmcplugin.h

```diff
@@ -103,9 +103,53 @@ namespace XBMCAddon
      *                           - SORT_METHOD_LABEL_IGNORE_THE, SORT_METHOD_VIDEO_SORT_TITLE,
      *                           - SORT_METHOD_VIDEO_SORT_TITLE_IGNORE_THE, SORT_METHOD_FULLPATH,
      *                           - SORT_METHOD_LABEL_IGNORE_FOLDERS, SORT_METHOD_CHANNEL
-     * 
+     *   - sortMethods available:
```

Comment:
Do we really want to add all of these here or just say `xbmcplugin.SORTMETHOD_*`? Because I'm pretty sure the next person will forget to update this comment.